scripts/src/models/translations.ts

```diff
@@ -0,0 +1,32 @@
+import { BaseEntity, Column, Entity, JoinColumn, ManyToOne, ObjectType, OneToMany, PrimaryColumn } from "typeorm";
+import { register as Register } from "./index";
+import { DeepPartial } from "typeorm/common/DeepPartial";
```

Comment:
as we talked about, i prefer that the npm modules be at the top, especially as it is an import from `typeorm` then put it with the other import from there